app/gui2/shared/ast/index.ts

```diff
@@ -68,3 +68,31 @@ export function subtreeRoots(module: Module, ids: Set<AstId>): Set<AstId> {
   }
   return roots
 }
+
+function unwrapGroup(ast: Ast) {
+  while (ast instanceof Group && ast.expression) ast = ast.expression
+  return ast
+}
```

Comment:
```suggestion

function unwrapGroups(ast: Ast) {
  while (ast instanceof Group && ast.expression) ast = ast.expression
  return ast
}
```
There may be more than one.